apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx

```diff
@@ -1,33 +1,24 @@
-import React, { ReactNode, useContext, useEffect, useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { initializeApp } from "firebase/app";
-import { getRemoteConfig, fetchAndActivate, RemoteConfig } from "firebase/remote-config";
-import { DEFAULT_FEATURES } from "@ledgerhq/live-common/featureFlags/index";
-import { Features } from "@ledgerhq/types-live";
-import reduce from "lodash/reduce";
-import snakeCase from "lodash/snakeCase";
+import { getRemoteConfig, fetchAndActivate, RemoteConfig, getValue } from "firebase/remote-config";
+import {
+  DEFAULT_FEATURES,
+  formatDefaultFeatures,
+  LiveConfig,
+} from "@ledgerhq/live-config/FeatureFlags/index";
+import type { FirebaseFeatureFlagsProviderProps as Props } from "@ledgerhq/live-config/FeatureFlags/index";
 import { getFirebaseConfig } from "~/firebase-setup";
 
+LiveConfig.init({
```

Comment:
Each client initialize the live config so that they can pass custom configurations if needed

Reply:
After some consideration, I move the LiveConfig.init to ledger-live-desktop/src/renderer/init.tsx for LLD and to ledger-live-mobile/src/index.tsx for LLM now.
I think it would be better. Let me know if you have a better idea.

Reply:
@lvndry I was talking about the file itself

@hzheng-ledger perfect